index.html

```diff
@@ -1395,33 +1397,33 @@ <h2>Benefits of JSON-LD and LD-Proofs</h2>
 extensions. This approach enables developers to easily experiment with new
 cryptographic signature mechanisms that support selective disclosure,
 zero-knowledge proofs, and post-quantum algorithms.
-          </dd>
-          <dt id="pf17b">
+            </dd>
+            <dt id="pf17b">
 PF17b: Supports signature chaining.
-          </dt>
-          <dd>
+            </dt>
+            <dd>
 A signature chain is an ordered set of signatures over a data payload. Use
 cases, such as cryptographic signatures applied to a notarized document,
 typically require a signature by the signing party and then an additional one
 by a notary to be made after the original signing party has made their
 signature. Linked Data Proofs, including Linked Data Signatures, natively
 support chains of signatures. JWTs only enable a single signature over a
 single payload.
-          </dd>
-          <dt id="pf18b">
+            </dd>
+            <dt id="pf18b">
 PF18b: Does not require pre-processing or post-processing.
-          </dt>
-          <dd>
+            </dt>
+            <dd>
 In order to encode a <a>verifiable credential</a> or a
 <a>verifiable presentation</a> in a JWT, an extra set of steps
 are required to convert the data to and from the JWT format. No such extra
 converstion step are required for <a>verifiable credentials</a> and
 <a>verifiable presentations</a> protected by LD-Proofs.
-          </dd>
-          <dt id="pf19b">
+            </dd>
+            <dt id="pf19b">
 PF19b: Canonicalization requires more than base-64 encoding.
-          </dt>
-          <dd>
+            </dt>
+            <dd>
 The JWT format utilizes a simple base-64 encoding format to generate the
 cryptographic hash of the data. The encoding format for LD-Proofs require
```

Comment:
`format for LD-Proofs require`
-> `format for LD-Proofs requires`
(add `s` to `require`)

Reply:
fixed in [9a0eaac](https://github.com/w3c/vc-imp-guide/pull/28/commits/9a0eaac6b21a3c4c4a8281809a1e809f781aadfc)